sym_api_client_python/processors/sym_message_parser.py

```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
     def get_stream_id(self, message_data):
         return message_data['stream']['streamId']
 
+    def get_tags(self, json_nodes, tag_type):
+        tags = []
+        if json_nodes:
+            for k, v in json_nodes.items():
+                if 'id' in v and len(v['id']) > 0 and v['id'][0]['type'] == self.MENTION_TYPE:
```

Comment:
the condition should be: `v['id'][0]['type'] == tag_type`, isn't it?

Reply:
do you think we need to check: `'type' in v['id'][0]`?

Reply:
Fixed.
For the question. I do not know the purpose of those conditions but I added them anyways to stay consistent with the Java SDK.